src/tutorial-panel/components/tutorial-list/index.tsx

```diff
@@ -65,20 +65,21 @@ export default function TutorialList({
           </InternalBox>
         </InternalSpaceBetween>
         <InternalSpaceBetween size="l">
-          <a
-            {...focusVisible}
-            href={downloadUrl}
-            target="_blank"
-            rel="noopener noreferrer"
-            className={styles['download-link']}
-            aria-label={i18nStrings.labelTutorialListDownloadLink}
-          >
-            <InternalIcon name="download" />
-            <InternalBox padding={{ left: 'xs' }} color="inherit" fontWeight="bold" display="inline">
-              {i18nStrings.tutorialListDownloadLinkText}
-            </InternalBox>
-          </a>
-
+          {downloadUrl && (
+            <a
+              {...focusVisible}
+              href={downloadUrl}
+              target="_blank"
+              rel="noopener noreferrer"
+              className={styles['download-link']}
+              aria-label={i18nStrings.labelTutorialListDownloadLink}
+            >
+              <InternalIcon name="download" />
+              <InternalBox padding={{ left: 'xs' }} color="inherit" fontWeight="bold" display="inline">
+                {i18nStrings.tutorialListDownloadLinkText}
+              </InternalBox>
+            </a>
+          )}
           {/*
```

Comment:
How about cleaning up commented code a bit? it does not seem like we will use it any time soon

Reply:
https://github.com/cloudscape-design/components/pull/883

Reply:
removed in a separate PR as the code is more than just these lines, and to allow us to revert when we want to add that feature

Reply:
Great, even more code than I expected